### DISCUSSION TITLE
Out of memory issues with the larger datasets.

### DISCUSSION THREAD

**Discussion by jisnajc** [perspective-dev/perspective#2038] [Q&A]
What is the max records size the perspective grid can load. For small datasets like 10k records its loading, but I am getting cors error due to the OOM when I am  trying to load 1 million records in arrow table format.

**Comment by sinistersnare** [comment]
Perspective runs locally on your machine. If you are getting an out-of-memory, you should get more memory.

I am not sure what you mean by a CORS error, that has to due with browser security. If you can post screenshots, or logs from your console, that would certainly help.

There is a template for creating an issue on this tracker, using it helps the maintainers of the project keep track of things.
